apps/web/src/pages/get-started/GetStartedPage.tsx

```diff
@@ -1,19 +1,30 @@
-import React from 'react';
-
 import { HeaderLayout } from './layout/HeaderLayout';
 import PageContainer from '../../components/layout/components/PageContainer';
 import PageHeader from '../../components/layout/components/PageHeader';
 import { GetStartedTabs } from './components/get-started-tabs/GetStartedTabs';
+import { useAuthContext } from '../../components/providers/AuthProvider';
+import { Center, Loader } from '@mantine/core';
+import { colors } from '@novu/design-system';
+import { useTabSearchParams } from './components/get-started-tabs/useGetStartedTabs';
 
 const PAGE_TITLE = 'Get started';
 
 export function GetStartedPage() {
+  const { currentOrganization } = useAuthContext();
+  const { currentTab, setTab } = useTabSearchParams();
```

Comment:
Thanks for the feedback! I will merge it as is for now, but we can memoize the tab behaviors so that they don't re-render the whole component tree if it becomes necessary.